### PR TITLE
Add test case to illustrate how to properly match against "any object"

### PR DESCRIPTION
**Pull request by ralflang**
It's not really about a bug in the trait - it works as designed.
However, you cannot import object to the current namespace and you must not qualify a type hint against object either.
This lead to a bug in my ListenerProvider implementation and I thought it would be handy to have this unit test as an illustration of how to use this edge case.

### CODE REVIEW

File: tests/InvokableAnyObject.php [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+namespace Fig\EventDispatcher;
+
+class InvokableAnyObject
+{
+    public function __invoke(object $event)
+    {
+        // do nothing
+    }
+}
```
Comment (Jean85) [inline tests/InvokableAnyObject.php]: End of file is missing.
```suggestion
}

```

**Review by Jean85 — CHANGES_REQUESTED** [review]

File: tests/InvokableAnyObject.php (hunk 1 above)
Comment (ralflang) [reply tests/InvokableAnyObject.php]: Fixed via commit - something blocks me from "accepting" the suggestion.

**Comment by ralflang** [comment]
Closing - discord suggested this doesn't achieve the intended heads up anf I should come up with another form of raising awareness.
